espanol/ejercito/ejercito.md

```diff
@@ -0,0 +1 @@
+Y vinieron un ejército de ángeles y se pusieron a tocar el violín
```

Comment:
I have made the story reacheable(i think i made it well) and push me changes into my changes branch